Source/Core/Mixin.lua

```diff
@@ -78,6 +78,32 @@ local function InitializeAuctionHouseTabs()
   end
 end
 
+local function InitializeSplashScreen()
+  -- TODO Check for display setting before creating
```

Comment:
Can't you just move the code that checks to the start of the function?

Reply:
I need the reference to the frame for some of the functionality, unless I move that into global methods